Makefile

```diff
@@ -227,6 +227,7 @@ endif
 
 db/Dockerfile:
 	envsubst < $@.in > $@
+	cp ci/scripts/duplicate-db.sh defaultcontent/duplicate-db.sh
```

Comment:
This is not the right Makefile rule for this.
Why not place it in the db/ folder so it's already in the Docker build context?